src/components/Dashboard/FilingHistoryList/FiledLabel.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="filed-label d-inline">
+  <div
+    v-if="!!filing"
+    class="filed-label d-inline"
+  >
```

Comment:
This is because the test suite tries to mount this component with "null" filing and expected "nothing" to be rendered.